Block/Adminhtml/Edit/Options/Options.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+namespace SnowIO\AttributeOptionCode\Block\Adminhtml\Edit\Options;
+
+use Magento\Catalog\Setup\CategorySetup;
+use SnowIO\AttributeOptionCode\Api\AttributeOptionCodeRepositoryInterface;
+
+class Options extends \Magento\Eav\Block\Adminhtml\Attribute\Edit\Options\Options
+{
+    /** @var AttributeOptionCodeRepositoryInterface */
+    private $attributeOptionCodeRepository;
+
+    /**
+     * @var string
+     */
+    protected $_template = 'SnowIO_AttributeOptionCode::catalog/product/attribute/options.phtml';
+
+    /**
+     * Options constructor.
+     * @param \Magento\Backend\Block\Template\Context $context
+     * @param \Magento\Framework\Registry $registry
+     * @param \Magento\Eav\Model\ResourceModel\Entity\Attribute\Option\CollectionFactory $attrOptionCollectionFactory
+     * @param \Magento\Framework\Validator\UniversalFactory $universalFactory
+     * @param AttributeOptionCodeRepositoryInterface $attributeOptionCodeRepository
+     * @param array $data
+     */
+    public function __construct(
+        \Magento\Backend\Block\Template\Context $context,
+        \Magento\Framework\Registry $registry,
+        \Magento\Eav\Model\ResourceModel\Entity\Attribute\Option\CollectionFactory $attrOptionCollectionFactory,
```

Comment:
You're mix and matching fully qualified class names with relative here, maybe just stick with `use` imports?